### PR TITLE
added ATC Code ETL appyter

### PR DESCRIPTION
**Pull request by ekropiwnicki**
Initial commit for Drugmonizome ATC Code ETL appyter

### CODE REVIEW

**Comment by ekropiwnicki** [comment]
Based on where the check failed, I'm guessing it is because the file needs to be manually imported by the user for the appyter to work. The file (https://www.drugbank.ca/releases/latest) requires an account to be downloaded from the website.

**Comment by u8sand** [comment]
You can still use the download link (which seems to be https://www.drugbank.ca/releases/5-1-7/downloads/all-full-database), it will fall back to directing the user to the site to download, and it will stop the test short.

File: appyters/ATC_Code_Drugmonizome_ETL/ATC_codes.ipynb [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+    "    section='data'\n",
+    ") %}\n",
+    "\n",
+    "{% set xml_file = FileField(\n",
+    "    constraint='.*\\.xml.zip$',\n",
+    "    name='atc codes', \n",
+    "    label='ATC Codes (xml.zip)', \n",
+    "    default='drugbank_all_full_database.xml.zip',\n",
+    "    examples={\n",
+    "        'drugbank_all_full_database.xml.zip': 'https://www.drugbank.ca/releases/5-1-7/downloads/all-full-database'\n",
```
Comment (ekropiwnicki) [inline appyters/ATC_Code_Drugmonizome_ETL/ATC_codes.ipynb:131]: I have the url path you referenced set within the example parameter of the FileField module, is there anything else I should have done?

Comment (u8sand) [reply appyters/ATC_Code_Drugmonizome_ETL/ATC_codes.ipynb:131]: @ekropiwnicki no this is a bug in the validator, I will fix it and merge your PR thanks

**Pull request merged**